### DISCUSSION TITLE
Can't get co-pilot to _not_ autocomplete on markdown files

### DISCUSSION THREAD

**Discussion by fakeharxy** [community/community#65577] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I use vimwiki a lot and find it very distracting when co-pilot autocompletes my thoughts there... 

I use neovim `init.lua`, I've tried adding the following:

```
vim.g.copilot_filetypes = { markdown = false, txt = false }
```

to no avail, co-pilot still adds suggestions to my `.md` vimwiki pages...

any further suggestions? Many thanks.

**Comment by geoffreylgv** [comment]
Hello @fakeharxy, 
Suggestion : you should deactivate copilot when you are working with Markdown.

**Reply by fakeharxy** [reply]
This isn't an ideal solution, I zip in and out of vim wiki all the time

**Comment by arafajs** [comment]
`vim.g.copilot_filetypes = {markdown = false}`

**Reply by fakeharxy** [reply]
Isn't this exactly what I wrote in my question?

**Comment by fakeharxy** [comment, marked as answer]
Hi, thanks for those who tried to assist... I figured out that vimwiki does not have a file type of markdown in lua... it has a file type of 'vimwiki'.

so 
```
vim.g.copilot_filetypes = { vimwiki = false }
```
is the solution.

**Reply by geoffreylgv** [reply]
Nice, thank you, @fakeharxy, for the update.
